social-network/src/Social_network.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'; //needed to test react components
+import ReactDOM from 'react-dom';
+import Article from './article'; 
+import Missed_article from './missedArticle';
+
+import ReactTestRenderer from 'react-test-renderer';
+import yourArticles from './yourArticles.json';
+
+
+
+describe('Article component', () => { 
+    it('renders without crashing', () => {
+        const div = document.createElement('div');
+        
+        const item = {
```

Comment:
You're pretty close with the article tests. The `title`, `description`, etc. should be nested under `article`. Ex.

```js
const props = {
  article: {
    title: '',
    ...
  }
};
```